### PR TITLE
Feature diploma/new tests

### PR DESCRIPTION
**Pull request by Lada-QA**

### CODE REVIEW

File: src/main/java/pages/RunsPage.java [hunk 1]
```diff
@@ -73,4 +100,34 @@ public String getSuccessMessage() {
...
+        chooseTestStatusAndClick.click();
+        log.info("Was input comment: " + textComment);
+        inputComment.sendKeys(textComment);
+        clickDropDownAssign.click();
+        chooseAndClickToAssign.click();
+        clickButtonEditResult.click();
+        return this;
+    }
+
+    public boolean testResultWasAdded() {
```
Comment (vchesnakoutest) [inline src/main/java/pages/RunsPage.java]: isTestResultAdded

File: src/test/resources/features/ui_tests/RunPage.feature [hunk 2]
```diff
@@ -9,4 +9,15 @@ Feature: Test runs
...
     And goes to the project and add new milestone 'This is Milestone'
     Then goes to the project and add new test run
-    And Verify test run was added
\ No newline at end of file
+    And Verify test run was added
+
+  Scenario: User adds new test and the test result
+    Given User send POST a new project 'This is a test project API' and announcement 'This is test project for API' using API
+    And User send POST request for add new suite 'This is a new test suite' with description 'Use the description to add additional context details' in the project using API
+    When User opens TestRail login page
```
Comment (vchesnakoutest) [inline src/test/resources/features/ui_tests/RunPage.feature]: And User opens TestRail login page

File: src/test/resources/features/ui_tests/RunPage.feature [hunk 3]
```diff
@@ -9,4 +9,15 @@ Feature: Test runs
...
+    And filling in standard email and password on Login page
+    When User clicks login button
```
Comment (vchesnakoutest) [inline src/test/resources/features/ui_tests/RunPage.feature]: And User clicks login button

File: src/test/resources/features/ui_tests/RunPage.feature [hunk 4]
```diff
@@ -9,4 +9,15 @@ Feature: Test runs
...
+    And User add section 'This is a test' and description 'This is description test' in project - This is a test project API
+    When User add new test case 'Result test' in section
```
Comment (vchesnakoutest) [inline src/test/resources/features/ui_tests/RunPage.feature]: And

File: src/test/resources/features/ui_tests/RunPage.feature [hunk 5]
```diff
@@ -9,4 +9,15 @@ Feature: Test runs
...
+    And goes to the runs page and add the test result
```
Comment (vchesnakoutest) [inline src/test/resources/features/ui_tests/RunPage.feature]: When

File: src/test/resources/features/ui_tests/RunPage.feature [hunk 6]
```diff
@@ -9,4 +9,15 @@ Feature: Test runs
     When User clicks login button
     And goes to the project and add new milestone 'This is Milestone'
     Then goes to the project and add new test run
-    And Verify test run was added
\ No newline at end of file
+    And Verify test run was added
```
Comment (vchesnakoutest) [inline src/test/resources/features/ui_tests/RunPage.feature]: Then

File: src/test/resources/features/ui_tests/RunPage.feature [hunk 7]
```diff
@@ -9,4 +9,15 @@ Feature: Test runs
     When User clicks login button
```
Comment (vchesnakoutest) [inline src/test/resources/features/ui_tests/RunPage.feature]: And

File: src/test/resources/features/ui_tests/RunPage.feature [hunk 8]
```diff
@@ -9,4 +9,15 @@ Feature: Test runs
...
     And goes to the project and add new milestone 'This is Milestone'
     Then goes to the project and add new test run
```
Comment (vchesnakoutest) [inline src/test/resources/features/ui_tests/RunPage.feature]: When

**Review by vchesnakoutest — CHANGES_REQUESTED** [review]

File: src/main/java/pages/RunsPage.java (hunk 1 above)
Comment (Lada-QA) [reply src/main/java/pages/RunsPage.java]: fixed

File: src/main/java/steps/CustomizationsSteps.java [hunk 9]
```diff
@@ -10,25 +10,17 @@
...
-        int targetStringLength = 10;
-        Random random = new Random();
-        StringBuilder buffer = new StringBuilder(targetStringLength);
-        for (int i = 0; i < targetStringLength; i++) {
-            int randomLimitedInt = leftLimit + (int)
-                    (random.nextFloat() * (rightLimit - leftLimit + 1));
-            buffer.append((char) randomLimitedInt);
-        }
-        String generatedString = buffer.toString();
+        createRandomName();
```
Comment (vchesnakoutest) [inline src/main/java/steps/CustomizationsSteps.java:23]: 👍

File: src/main/java/utils/Random.java [hunk 10]
```diff
@@ -0,0 +1,23 @@
+package utils;
+
+public class Random {
+
+    public static String RANDOM_NAME;
+
+    public Random() {
```
Comment (vchesnakoutest) [inline src/main/java/utils/Random.java]: private, т.к метод утилитный

**Review by vchesnakoutest — CHANGES_REQUESTED** [review]

**Pull request merged**